.gitignore

```diff
@@ -56,6 +56,7 @@ cover/
 *.pot
 
 # Django stuff:
+*.log
```

Comment:
Group project specific override files under a separate section

Reply:
removed unwanted `.gitignores` and created project specific section

Reply:
Please don't remove anything from the predefined `.gitignore` file. It is default from GitHub.